### ISSUE TITLE
verbosity attribute in MediaTrace

### ISSUE THREAD

**Issue by dericed** [MediaArea/MediaConch_SourceCode#288]
Some implementation checks require --ParseSpeed 1 or other verbose MediaTrace options, such as testing to make sure all Seek elements resolve (this fails on low verbosity mediatrace, since not all Clusters are noted).

I propose a `verbose` attribute to be added where ever the `parser` attribute is used, so in `block`, `ref`, and `MediaTrace`.

**Comment by JeromeMartinez** [comment]
I don't like we terms `ParseSpeed`(no real meaning) and `verbosity`(it is not about verbosity but how much data we check).

`  completeness="full", `  completeness="fast", `  completeness="medium"?
(I don't like the idea of having numbers, is "1" full or fast and is "9" full or fast?)

**Comment by dericed** [comment]
you're right, i remember we debate this before. I was on the side of `coverage`

**Comment by JeromeMartinez** [comment]
ho, I forgot the debate.

`  coverage="full"` (--ParseSpeed=1), `  coverage="fast"` (--ParseSpeed=0), `  coverage="medium"` (--ParseSpeed=0.5 and default)?

**Comment by dericed** [comment]
I suggest number with coverage with words as aliases to numbers.

**Comment by JeromeMartinez** [comment]
which number? which alias?

**Comment by JeromeMartinez** [comment]
I prefer something unique (number or text), having 2 methods for saying the same thing may be confusing.

**Comment by dericed** [comment]
Check the man page for the flac utility. There is something like this in the compression option.

**Comment by JeromeMartinez** [comment]
we talk of an XML output, so a parser will have to support both if we accept both. This is not similar to command line options
